Sources/SwiftDocC/Infrastructure/Communication/WebKitCommunicationBridge.swift

```diff
@@ -49,10 +49,8 @@ public struct WebKitCommunicationBridge: CommunicationBridge {
         do {
             let encodedMessage = try JSONEncoder().encode(message)
             let messageJSON = String(data: encodedMessage, encoding: .utf8)!
-                // Escape backticks to avoid conflicting with JavaScript's template string syntax.
-                .replacingOccurrences(of: "`", with: "\\`")
             
-            evaluateJavaScript("window.bridge.receive(JSON.parse(`\(messageJSON)`))") { _, _ in }
+            evaluateJavaScript("window.bridge.receive(JSON.parse(JSON.stringify(\(messageJSON))))") { _, _ in }
```

Comment:
Doesn't this mean that what we're passing to `window.bridge.receive` is a string instead of an object now? Since DocC-Render expects an object, I suspect this change will break things?

Reply:
No. `JSON.prase` parses it to an object.

Reply:
Aah I see, the confusion for me was that `messageJSON` isn't actually a JavaScript string literal, it's a JSON object. In that case why not pass it directly with `window.bridge.receive(\(messageJSON))`?

Reply:
was just about to ask :)

Reply:
Ah right good point... Curious why that wasn't done originally then? But yes that seems to work as well – I'll update the PR.